django/urls/converters.py

```diff
@@ -0,0 +1,48 @@
+from django.conf import settings
+from django.utils import lru_cache
+
+
+class BaseConverter(object):
+    pass
+
+
+class IntConverter(BaseConverter):
+    regex = '[0-9]+'
```

Comment:
Seems negative floats and ints ["come up quite often"](https://github.com/pallets/flask/issues/315#issuecomment-2007279) in issues in flask, but that they still believe that the default should be to disallow them. I tend to agree that most people probably would only think of positive ints when specifying that in the URL, so I'm with @aaugustin here.